test/src/components/issues/Issue11.js

```diff
@@ -1,7 +1,6 @@
-/* @flow */
```

Comment:
test lib is ignored in `.flowconfig`, but causes false-positive errors when included in users config

Reply:
it also probably makes sense to add `test` lib to `.npmignore` to avoid publishing it to npm at all